pkg/engine/engine.go

```diff
@@ -280,6 +281,12 @@ func NewEngine(ctx context.Context, cfg *Config) (*Engine, error) {
 
 		_, ok = results["filtered_unverified"]
 		engine.logFilteredUnverified = ok
+
+		// If retain_false_positives is set, we also need to notify unverified results.
+		if _, ok = results["retain_false_positives"]; ok {
```

Comment:
The other options you can choose are nouns, so I think this should be a noun too for consistency. (Probably just `false_positives`.)

Reply:
Hmm, I agree with maintaining consistency, but `false_positives` _feels_ very vague. Is the assumption that any key in the results map implicitly means we keep the result?